### PR TITLE
Release v0.15.0-rc.0

### PR DESCRIPTION
**Pull request by ericnordelo**

### CODE REVIEW

File: docs/modules/ROOT/pages/utils/_class_hashes.adoc [hunk 1]
```diff
@@ -1,13 +1,13 @@
 // Version
-:class-hash-cairo-version: https://crates.io/crates/cairo-lang-compiler/2.6.3[cairo 2.6.3]
+:class-hash-cairo-version: https://crates.io/crates/cairo-lang-compiler/2.6.5[cairo 2.6.5]
```
Comment (andrew-fleming) [inline docs/modules/ROOT/pages/utils/_class_hashes.adoc:2]:
```suggestion
:class-hash-cairo-version: https://crates.io/crates/cairo-lang-compiler/2.6.4[cairo 2.6.4]
```
Scarb went rogue

File: CHANGELOG.md [hunk 2]
```diff
@@ -8,6 +8,17 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+
+### Changed
+
+- `Trace`, `Checkpoint`, and `StorageArray` structs made public.
+
+### Changed (Breaking)
+
+- Removed `num_checkpoints` and `checkpoints` from `ERC20VotesABI`.
+
+
```
Comment (andrew-fleming) [inline CHANGELOG.md:21]:
```suggestion

```

File: docs/modules/ROOT/pages/utils/_class_hashes.adoc (hunk 1 above)
Comment (ericnordelo) [reply docs/modules/ROOT/pages/utils/_class_hashes.adoc:2]: Good catch 👍🏼

**Review by andrew-fleming — APPROVED** [review]
LGTM!

**Pull request merged**